src/d3a/d3a_core/rq_job_handler.py

```diff
@@ -31,11 +31,7 @@ def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_
         GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
         d3a.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
     saved_state = decompress_and_decode_queued_strings(saved_state)
-    log.error(f"Scenario: {scenario}\n"
-              f"Settings: {settings}\n"
-              f"Events: {events}\n"
-              f"Aggregator device mapping: {aggregator_device_mapping}\n"
-              f"Previous simulation state: {saved_state}")
```

Comment:
I assume the intuition may be that these info already exist in DB, so no need to clutter the logs with redundant infos

Reply:
This is part of @hannesdiedrich's D3ASIM-3451. The changes ended up here when I targeted `staging` instead of `master`, but we decided to keep them and merge them together as part of this PR.